xchange-cryptowatch/pom.xml

```diff
@@ -0,0 +1,40 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.knowm.xchange</groupId>
+        <artifactId>xchange-parent</artifactId>
+        <version>4.4.2-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>xchange-cryptowatch</artifactId>
+
+    <name>XChange Kraken</name>
```

Comment:
not Kraken

Reply:
ops, sorry.
I will fix the name
Cryptowatch is owned by kraken anyway.